File: FrontEnd/src/app/models/usuario.ts
export class usuario {

    idUsuario:number;
    nombre: string;
    apellido: string;
    dni: number;
    email: string;
    password: string;
    fechaNacimiento: Date;
    fechaAlta: Date;
    idCuenta: number;





    constructor(idUsuario:number,nombre: string, apellido: string, email: string, password1: string, dni: number, fechanacimiento: Date,fechaAlta: Date,idCuenta: number) {
        this.idUsuario = idUsuario; 
        this.nombre = nombre;
        this.apellido = apellido;
        this.dni = dni;
        this.email = email;
        this.password = password1;
        this.fechaNacimiento = fechanacimiento;
        this.fechaAlta=fechaAlta;
        this.idCuenta=idCuenta;

    }

}

File: BackEnd/CryptoPIL/Negocio/Usuario.cs
using System;
using System.Collections.Generic;

namespace CryptoPIL.Models
{
    public partial class Usuario
    {
        public int IdUsuario { get; set; }
        public string Nombre { get; set; } = null!;
        public string Apellido { get; set; } = null!;
        public int Dni { get; set; }
        public string Email { get; set; } = null!;
        public string Password { get; set; } = null!;
        public DateTime FechaNacimiento { get; set; }
        public DateTime FechaAlta { get; set; }
        public DateTime? FechaBaja { get; set; }
        public int IdCuenta { get; set; }

        public virtual Cuenta IdCuentaNavigation { get; set; } = null!;
    }
}
